src/shared/ui/components/widget-tab/tab-handle.component.tsx

```diff
@@ -0,0 +1,39 @@
+import { classes } from '../../utils/classes';
+
+import { TabHandleProperties } from './tab-handle.types';
+
+export const TabHandle = ({
+  left,
+  imageSrc,
+  name,
+  isActive,
+  theme,
+  onClick,
+}: TabHandleProperties) => {
+  return (
+    <div
+      onClick={onClick}
+      style={{
+        left,
+      }}
```

Comment:
Currently we calculate `left` property based on index, I was thinking if this would be possible by having a single div wrapping all `TabHandles`, this div would have absolute position, same as this TabHandle and it would just display all Tabs inside of it, by doing so you don't need to calculate `left` property as they will just be aligned in a single row, probably with addition of `display: flex`, etc. WDYT?

Reply:
I was also considering that, but I wasn't sure where to place the wrapper since we don't have a higher-level component that groups all the widgets together